docs/faq.rst

```diff
@@ -193,7 +193,8 @@ Why are my workflows from an organization I belong to not visible?
 Organizations have the ability to restrict access to the API for third
 party applications. GitHub provides a
 `tutorial <https://help.github.com/en/articles/enabling-oauth-app-access-restrictions-for-your-organization/>`__
-on how to add these restrictions to your organizations.
+on how to add these restrictions to your organizations.  For recently-created GitHub organizations,
```

Comment:
Wasn't aware of this, do you have a handy link to read up on this?

Reply:
> Wasn't aware of this, do you have a handy link to read up on this?

See the first paragraph in body-sized type:
https://docs.github.com/en/organizations/managing-oauth-access-to-your-organizations-data/enabling-oauth-app-access-restrictions-for-your-organization
I tested this a long time ago, so I did it again just now to make sure. Indeed, in a fresh, new, untouched dummy GitHub org, indeed, the restrictions are enabled by default:
![image](https://github.com/dockstore/dockstore-documentation/assets/88108675/04f41197-2b0d-4a5a-b8b5-ab714578a22e)

Reply:
Annoying change, but thanks!